components/profile_picture.jsx

```diff
@@ -72,7 +72,7 @@ export default class ProfilePicture extends React.PureComponent {
             <span className='status-wrapper'>
                 <img
                     className='more-modal__image'
-                    alt={`${this.props.user.username || 'user'} profile image`}
+                    alt={'user profile image'}
```

Comment:
Note `this.props.user` being tested as `truthy` on https://github.com/mattermost/mattermost-webapp/blob/0e1c2384bd26dfd5bd7891ada9acabd06bbc4881/components/profile_picture.jsx#L37.